lab_1_classify_by_unigrams/main.py

```diff
@@ -5,107 +5,62 @@
-
-    Args:
-        text (str): A text
-
-    Returns:
-        list[str] | None: A list of lower-cased tokens without punctuation
-
-    In case of corrupt input arguments, None is returned
-    """
-
-
-def calculate_frequencies(tokens: list[str] | None) -> dict[str, float] | None:
-    """
-    Calculate frequencies of given tokens.
-
-    Args:
-        tokens (list[str] | None): A list of tokens
-
-    Returns:
-        dict[str, float] | None: A dictionary with frequencies
-
-    In case of corrupt input arguments, None is returned
-    """
-
-
-def create_language_profile(language: str, text: str) -> dict[str, str | dict[str, float]] | None:
-    """
-    Create a language profile.
-
-    Args:
-        language (str): A language
-        text (str): A text
-
-    Returns:
-        dict[str, str | dict[str, float]] | None: A dictionary with two keys – name, freq
-
-    In case of corrupt input arguments, None is returned
-    """
-
-
-def calculate_mse(predicted: list, actual: list) -> float | None:
-    """
-    Calculate mean squared error between predicted and actual values.
-
-    Args:
-        predicted (list): A list of predicted values
-        actual (list): A list of actual values
-
-    Returns:
-        float | None: The score
-
-    In case of corrupt input arguments, None is returned
-    """
-
-
-def compare_profiles(
-    unknown_profile: dict[str, str | dict[str, float]],
-    profile_to_compare: dict[str, str | dict[str, float]],
-) -> float | None:
-    """
-    Compare profiles and calculate the distance using symbols.
-
-    Args:
-        unknown_profile (dict[str, str | dict[str, float]]): A dictionary of an unknown profile
-        profile_to_compare (dict[str, str | dict[str, float]]): A dictionary of a profile
-            to compare the unknown profile to
-
-    Returns:
-        float | None: The distance between the profiles
-
-    In case of corrupt input arguments or lack of keys 'name' and
-    'freq' in arguments, None is returned
-    """
-
-
-def detect_language(
-    unknown_profile: dict[str, str | dict[str, float]],
-    profile_1: dict[str, str | dict[str, float]],
-    profile_2: dict[str, str | dict[str, float]],
-) -> str | None:
-    """
-    Detect the language of an unknown profile.
-
-    Args:
-        unknown_profile (dict[str, str | dict[str, float]]): A dictionary of a profile
-            to determine the language of
-        profile_1 (dict[str, str | dict[str, float]]): A dictionary of a known profile
-        profile_2 (dict[str, str | dict[str, float]]): A dictionary of a known profile
-
-    Returns:
-        str | None: A language
-
-    In case of corrupt input arguments, None is returned
-    """
-
+def tokenize(text):
+
+    text = text.lower()
+    split_text = ''.join(text)
+    token_list = []
+    for symbol in split_text:
+        if symbol.isalpha():
+            token_list.append(symbol)
+    return token_list
+
+def calculate_frequencies(token_text):
+
```

Comment:
add checks for variable type, all inputs may be corrupted

Reply:
done, thanks